lib/xdr/ledger_entries/config_setting_ledger_key.ex

```diff
@@ -0,0 +1,56 @@
+defmodule StellarBase.XDR.ConfigSettingLedgerKey do
```

Comment:
I think the module should be named `LedgerKeyConfigSetting`, since this is the struct ConfigSetting inside LedgerKey union